integrations/electricity/energinet.js

```diff
@@ -149,7 +152,7 @@ async function collect(state = {}, logger) {
   const activities = Object.entries(
     groupBy(timeSeries, (dataPoint) => moment(dataPoint.datetime).startOf('day').toISOString()) // regroup by day
   ).map(([k, values]) => ({
-    id: `energinet${values[0].mRID}${k}`,
+    id: `energinet-${Array.from(new Set(values.map((v) => v.mRID))).sort()}-${k}`,
```

Comment:
If I understood this correctly, then this will convert an array to string and be part of the `id`. Is that intended? Shouldn't we use the id of the particular metering point instead of the array?

Reply:
clarified by last comment